src/tooling/piral-cli-webpack5/src/helpers.ts

```diff
@@ -2,19 +2,31 @@ import { existsSync } from 'fs';
 import { Configuration } from 'webpack';
 import { DefaultConfiguration } from './webpack/common';
 
-export function extendConfig(
+export async function extendConfig(
   [webPackConfig, enhancer]: DefaultConfiguration,
   otherConfigPath: string,
   overrides: Configuration = {},
-): Configuration {
+): Promise<Configuration> {
   const original = webPackConfig;
 
   if (existsSync(otherConfigPath)) {
     try {
-      const otherConfig = require(otherConfigPath);
+      let otherConfig = require(otherConfigPath);
```

Comment:
Question here: Would it make sense to check / support `mjs` extension here? And maybe use `import(...)` instead of `require(...)` when an `mjs` file is found?